### PR TITLE
chore: add issue & pr templates

### LINKED ISSUE #29: Add issue and PR templates
Maybe also add them for eludris/pilfer and eludris/pengin? 

also maybe maybe add dumbified versions for docs and elusite??? :lol:

### PR DESCRIPTION
**Pull request by spifory**
## Description

This adds Issue & Pull Request templates that are copied from [eludris/eludris](https://github.com/eludris/eludris).

- Closes eludris/eludris#29

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+<!-- Link any issues if applicable.
+Use keywords from https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests
+-->
+
+<!-- Uncomment based on the type of your changes below -->
+
+<!--
+## This is a **Content Change**
+- [ ] Changes have been tested.
```
Comment (EnokiUN) [inline .github/PULL_REQUEST_TEMPLATE.md]: why would I test that tho lol

**Review by EnokiUN — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (spifory) [reply .github/PULL_REQUEST_TEMPLATE.md]: https://github.com/eludris/elusite/pull/2/commits/5f483daff42f5e566890794bbd4dab0e411695b1

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+<!-- Explain what this Pull Request changes -->
+
+<!-- Link any issues if applicable.
+Use keywords from https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests
+-->
+
+<!-- Uncomment based on the type of your changes below -->
+
+<!--
```
Comment (EnokiUN) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: Maybe remove this entire thing since it'd be obvious from the pr description

**Review by EnokiUN — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (toolifelesstocode) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: https://github.com/eludris/elusite/pull/2/commits/1233f1fe67d833ea62951ab8e1d959e6fecce00e

Comment (toolifelesstocode) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: https://github.com/eludris/elusite/pull/2/commits/1233f1fe67d833ea62951ab8e1d959e6fecce00e

**Pull request merged**